### ISSUE TITLE
Does Loggie plan to join CNCF？

### ISSUE THREAD

**Issue by LronDC** [loggie-io/loggie#150]
Want to know about Loggie's roadmap

**Comment by ethfoo** [comment]
Yes, we planned to contribute Loggie to CNCF as sandbox project, but there is still a lot of work.

RoadMap:
https://github.com/orgs/loggie-io/projects/1
https://loggie-io.github.io/docs/getting-started/roadmap/roadmap-2022/

**Comment by LronDC** [comment]
Thank you, good to know that.
